@kiva/kv-tokens/primitives.js

```diff
@@ -512,9 +512,12 @@ export default {
 	},
 	shadows: {},
 	radii: {
-		sm: 4,
+		xs: 4,
+		sm: 8,
+		md: 12,
 		default: 16,
-		lg: 40,
+		lg: 16,
```

Comment:
These updates were based on an initial scale provided by design which we can see duplciates our default as lg. I discussed making this distinct with design earlier today so we may set `lg` to something else, either 20 or 24...